test/inductor/test_extension_backend.py

```diff
@@ -50,12 +49,7 @@
 
 
 def remove_build_path():
-    if sys.platform == "win32":
-        # Not wiping extensions build folder because Windows
-        return
-    default_build_root = torch.utils.cpp_extension.get_default_build_root()
-    if os.path.exists(default_build_root):
-        shutil.rmtree(default_build_root, ignore_errors=True)
+    torch.utils.cpp_extension.remove_default_build_root()
```

Comment:
You can use `torch/testing/_internal/*` to host this kind of utilities.
Also I would suggest remove this indirection since it just calls down the same function?